pyethapp/app.py

```diff
@@ -1,3 +1,11 @@
+import sys
+req_version = (2,7,9)
+cur_version = sys.version_info
+
+if cur_version < req_version:
+    print "Your Python interpreter is too old. Please consider upgrading."
```

Comment:
reporting the required version would be nice. also `required_version` would be a better name.